crates/turborepo-lib/src/commands/link.rs

```diff
@@ -31,13 +33,19 @@ pub(crate) enum SelectedTeam<'a> {
     Team(&'a Team),
 }
 
+#[derive(Clone)]
+pub(crate) enum SelectedSpace<'a> {
+    Space(&'a Space),
+}
+
 pub(crate) const REMOTE_CACHING_INFO: &str = "  Remote Caching shares your cached Turborepo task \
                                               outputs and logs across
   all your team’s Vercel projects. It also can share outputs
   with other services that enable Remote Caching, like CI/CD systems.
   This results in faster build times and deployments for your team.";
 pub(crate) const REMOTE_CACHING_URL: &str =
     "https://turbo.build/repo/docs/core-concepts/remote-caching";
+pub(crate) const SPACES_URL: &str = "https://vercel.com/docs/spaces";
```

Comment:
I think this should be a turbo.build URL, not vercel. The feature to post to a space belongs to turbo

Reply:
Yep, this is a placeholder for now